deployment/submit_emr_job_train.py

```diff
@@ -23,57 +23,56 @@
 from time import gmtime, strftime
 import daiquiri
 
-import recommendation_engine.config.cloud_constants as config
+# import recommendation_engine.config.cloud_constants as config
 
 daiquiri.setup(level=logging.DEBUG)
 _logger = daiquiri.getLogger(__name__)
 
 COMPONENT_PREFIX = "chester"
 
 
-def submit_job(input_bootstrap_file, input_src_code_file):
+def submit_job():
     """Submit new job with specified parameters."""
     str_cur_time = strftime("%Y_%m_%d_%H_%M_%S", gmtime())
-
     # S3 bucket/key, where the input spark job ( src code ) will be uploaded
-    s3_bucket = config.DEPLOYMENT_PREFIX + '-automated-analytics-spark-jobs'
-    s3_key = '{}_{}_tf_training_job.zip'.format(config.DEPLOYMENT_PREFIX, COMPONENT_PREFIX)
+    s3_bucket = 'cvae-insights'
```

Comment:
Pick this and the s3_key from the environment, don't hardcode it.

Reply:
Ok. Actually, this is just for testing purposes, the Training job, and the Bootstrap file. But yeah its good practice to pick from the environment, will update accordingly.